issue98-websockets/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "issue98-websockets",
+  "version": "1.0.0",
+  "description": "Building a chat using WebSockets",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/server.js",
+    "dev": "nodemon src/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/picolloo/CurriculumExpansion.git"
```

```suggestion
    "url": "git+https://github.com/freeCodeCamp/CurriculumExpansion.git"
```
I guess we'd be updating all these URLs